sdk/storage/Azure.Storage.Common/src/ClientsideEncryptionOptions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Cryptography;
+
+namespace Azure.Storage
+{
+    /// <summary>
+    /// Provides the client configuration options for connecting to Azure Blob using clientside encryption.
+    /// </summary>
+    public class ClientSideEncryptionOptions
```

Comment:
Are we adding flag that controls whether exception is thrown or not if client can't decrypt separately?

Reply:
Yes. Not pushed yet, but I'm almost done testing.